packages/dev/docs/src/docs.css

```diff
@@ -39,6 +39,7 @@ html, body {
   padding: 0 24px;
   background: var(--spectrum-global-color-gray-75);
   box-sizing: border-box;
+  overflow-y: scroll;
```

Comment:
remove height: 100% from this element and add bottom: 0;

also, in spectrum-SideNav, remove height 100%

should fix the enormous gap at the bottom

Reply:
I made https://github.com/adobe-private/react-spectrum-v3/pull/422 with your advice.

Reply:
you have overflow-y here twice 😉